src/lib.rs

```diff
@@ -47,9 +53,9 @@ pub fn run_compiled(book: &Book, mem_size: usize) -> (Net, RunStats) {
 }
 
 pub fn run_book(mut book: DefinitionBook, mem_size: usize) -> anyhow::Result<(Term, DefNames, RunInfo)> {
-  let compiled = compile_book(&mut book)?;
+  let (compiled, id_map) = compile_book(&mut book)?;
```

Comment:
Not a big fan of the name id_map, since we already have pretty much the same thing inside the definition book, but there it's DefId to hvm-lang name, while here it's DefId to hvmc name.
`id_map` feels like it could be either, so a bit confusing.